css/amp-story-player.css

```diff
@@ -29,6 +29,10 @@ amp-story-player a:first-of-type {
   display: block;
 }
 
+amp-story-player a:not(:first-of-type) {
```

Comment:
Is this new code?

Reply:
Yep, I'm hiding them because otherwise they stack on top of the first story anchor tag whenever the script hasn't finish loaded yet (pre-fetch experience).